### ISSUE TITLE
Focus gets lost after hitting keyboard ENTER on the simulator play button

### ISSUE THREAD

**Issue by microbit-grace** [microsoft/pxt-microbit#6406]
The location of the focus is unclear to the user after hitting keyboard "ENTER" on the simulator play button.

**To Reproduce**
1. Go to https://makecode.microbit.org/beta#editor.
2. Keyboard navigate to the simulator play/stop button.
3. Hit "ENTER" on the play button (You may need to hit "ENTER" on the stop button first to see the play button).
4. See the focus get lost.

It seems the focus gets moved to the simulator iframe, which does not have a visible focus ring. 

The expected behaviour could be to show the same focus ring as when the simulator board is focused (outline on the boardview div), or to keep the focus on the play/stop button.

**Comment by microbit-matt-hillsdon** [comment]
If we think moving to the sim is the right idea (vs staying on the button) then moving to #boardview would give a focus outline and match where the tab stop is for the normal tab sequence.

**Comment by microbit-matt-hillsdon** [comment]
This was fixed by https://github.com/microsoft/pxt/pull/10734

Closing.
